src/features/TrendingMovies/components/ImageSlider.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import ScheduleIcon from '@mui/icons-material/Schedule';
+
+interface Movie {
+    title: string;
+    imageUrl: string;
+    description: string;
+    duration: number;
+    genre: string;
+}
+
+interface ImageSliderProps {
+    movies: Movie[];
+}
+
+const ImageSlider = ({ movies }: ImageSliderProps) => {
```

Comment:
Let's rename it as MovieSlider, I've just realised that ImageSlider is not right name for component which renders info about movie but not only image + by name it as MovieSlider we show for everyone that this component is related with movies